addons/vehicle_damage/functions/fnc_medicalDamage.sqf

```diff
@@ -0,0 +1,44 @@
+#include "..\script_component.hpp"
+/*
+ * Author: johnb43
+ * Applies medical damage to a unit.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ * 1: Source <OBJECT>
+ * 2: Instigator <OBJECT>
+ * 3: Guarantee death? <BOOL> (default: false)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorObject, player, player] call ace_vehicle_damage_fnc_medicalDamage;
+ *
+ * Public: No
+ */
+
+params ["_unit", "_source", "_instigator", ["_guaranteeDeath", false]];
+
+// Check if unit is invulnerable
+if !(isDamageAllowed _unit && {_unit getVariable [QEGVAR(medical,allowDamage), true]}) exitWith {};
+
+if (["ace_medical"] call EFUNC(common,isModLoaded)) then {
+    for "_i" from 0 to floor (4 + random 3) do {
+        [_unit, random [0, 0.66, 1], selectRandom ["Head", "Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"], selectRandom ["bullet", "shell", "explosive"], _instigator] call EFUNC(medical,addDamageToUnit);
+    };
+} else {
+    {
+        _unit setHitPointDamage [_x, (_unit getHitPointDamage _x) + random [0, 0.66, 1], true, _source, _instigator];
+    } forEach ["HitFace", "HitNeck", "HitHead", "HitPelvis", "HitAbdomen", "HitDiaphragm", "HitChest", "HitBody", "HitArms", "HitHands", "HitLegs"];
+};
+
+// If guaranteed death is wished
+if (_guaranteeDeath && {alive _unit}) then {
+    // From 'ace_medical_status_fnc_setDead': Kill the unit without changing visual appearance
```

Comment:
We can, but it would either create a medical dependency or we would have to add code both cases where medical is loaded or not. It would ultimately boil down to this code in the second case, so not worth it imo.
*If* we were to add API, it would have to be in common, in case medical isn't loaded.